### PR TITLE
add: standard guide style.

### PR DESCRIPTION
**Pull request by mishelashala**
The code is incongruent and inconsistent.

### CODE REVIEW

**Comment by pel-daniel** [comment]
Thanks for the PR.
I agree that a congruent style is needed throughout the project, but I don't find some of the rules of standardjs particular pleasant to me.

The most important thing on any format to me, is code that is easy to read.
Could you elaborate a bit why you recommend using it, and is standardjs well adopted by the js community?

**Comment by mishelashala** [comment]
I don't like standard JS. But is the standard by de facto. I prefer semistandard, is the same shit but with semicolons.

File: src/components/Brick.js [hunk 1]
```diff
@@ -14,19 +14,19 @@ import {
...
 
-  startDrag(mouseEvent) {
+  startDrag (mouseEvent) {
     const { handleMouseDown, id, position } = this.props
 
     handleMouseDown(id, mouseEvent, position)
   }
 
-  render() {
+  render () {
```
Comment (pel-daniel) [inline src/components/Brick.js:29]: There are 2 rules that I don't like. The first is this space (useless in my opinion)

File: src/components/Brick.js [hunk 2]
```diff
@@ -43,33 +43,34 @@ class Brick extends Component {
 
     return (
       <Group
-        onMouseDown={ this.startDrag }
-        y={ slotHeight }
+        onMouseDown={this.startDrag}
+        y={slotHeight}
```
Comment (pel-daniel) [inline src/components/Brick.js:47]: The second is this removed spaces.

**Pull request closed without merging**